src/shared/bus-unit-util.c

```diff
@@ -106,7 +107,18 @@ DEFINE_BUS_APPEND_PARSE("i", ioprio_class_from_string);
 DEFINE_BUS_APPEND_PARSE("i", ip_tos_from_string);
 DEFINE_BUS_APPEND_PARSE("i", log_facility_unshifted_from_string);
 DEFINE_BUS_APPEND_PARSE("i", log_level_from_string);
-DEFINE_BUS_APPEND_PARSE("i", parse_errno);
+
+static int parse_errno_or_action(const char *p) {
```

Comment:
i think it would be good to move this to seccomp-util.h or so, and given this a clearer name, maybe `seccomp_parse_errno_or_action()`, i.e. make clear that this is about seccomp